lib/web_ui/lib/src/engine/window.dart

```diff
@@ -27,14 +27,14 @@ class EngineWindow extends ui.Window {
 
   @override
   double get devicePixelRatio => _debugDevicePixelRatio != null
```

Comment:
could this be `double get devicePixelRatio => _debugDevicePixelRatio ?? browserDevicePixelRatio;`

Reply:
Done.